.gitignore

```diff
@@ -155,3 +155,8 @@ venv
 
 # tools
 tools/prometheus*
+
+# java build
+build/java/*
+java/third_party/*
```

Comment:
1. `build/java/` isn't needed anymore, because we've deprecated cmake.
2. we should exclude `java/third_party/workspace.bzl` and only ignore other files.

Reply:
`java/third_party/*` has already excluded `java/third_party/workspace.bzl` and `java/third_party/BUILD`. It only ignores the folders under `java/third_party`.